File: Hjemmesentral/app/src/main/java/com/homecentral/jrs/hjemmesentral/model/yr/Meta.java
package com.homecentral.jrs.hjemmesentral.model.yr;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "meta", strict = false)
public class Meta {

    @Element(name = "lastupdate")
    private String lastUpdate;

    @Element(name = "nextupdate")
    private String nextUpdate;
}

File: Hjemmesentral/app/src/main/java/com/homecentral/jrs/hjemmesentral/model/yr/Meta.kt
package com.homecentral.jrs.hjemmesentral.model.yr

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "meta", strict = false)
class Meta @JvmOverloads constructor(
    @field:Element(name = "lastupdate") private var lastUpdate: String = "",
    @field:Element(name = "nextupdate") private var nextUpdate: String = ""
)
